linker/jvm/src/main/scala/org/scalajs/linker/PathIRFile.scala

```diff
@@ -30,8 +31,11 @@ object PathIRFile {
       .map(new PathIRFileImpl(path, _))
   }
 
+  private[linker] def fileTimeToVersion(time: FileTime): ir.Version =
+    ir.Version.fromBytes(time.toString().getBytes(StandardCharsets.US_ASCII))
```

Comment:
That seems unnecessarily inefficient. Consider something like:
```scala
val instant = time.toInstant()
ir.Version.fromLongs(instant.getEpochSecond(), instant.getNano())
```

Reply:
I have. But that isn't lossless according to this: https://docs.oracle.com/javase/8/docs/api/java/nio/file/attribute/FileTime.html#toInstant--

`toString` is the only lossless serialization of `FileTime` I could find.

Reply:
Hum, OK. Probably worth a comment, then.